reconcile/templating/lib/merge_request_manager.py

```diff
@@ -121,34 +121,26 @@ def process(self, gitlab_cli: GitLabApi) -> None:
                 )
 
 
-class MrData(BaseModel):
-    data: list[TemplateOutput]
-    auto_approved: bool
-
-
 class MergeRequestManager(MergeRequestManagerBase[TemplateInfo]):
     def __init__(self, vcs: VCS, parser: Parser):
         super().__init__(vcs, parser, TR_LABEL)
 
-    def create_merge_request(self, data: MrData) -> None:
+    def create_merge_request(
+        self, data: TemplateResult, auto_approved: bool = False
+    ) -> None:
```

Comment:
in `MergeRequestManagerBase`, `create_merge_request` is an abstract method, better to keep params (`data`) unchanged.